update index page (#9)
process-engine/getting-started: docs/index.md

![BPMN Studio: Ausführen per Klick](images/bpmn-studio-design-play.png)

> Prozesse können mit individuellen Parametern gestartet werden. An exklusives Gateway kann ein Parameter übergeben werden, um das Prozessmodell auf einem bestimmten Pfad zu leiten. Diese Parameter können als JSON-Format in dem Feld angegeben werden.

Author: ```suggestion
Prozesse können und müssen in gewissen Fällen mit individuellen Parametern gestartet werden. In diesem Beispiel wird erwartet, dass die Eigenschaft "shoppingCardAmount" vom Aufrufer vorgegeben wird. Die Eigenschaft wird an einem exklusiven Gateway ausgewertet, um das Prozessmodell auf einem bestimmten Pfad zu leiten. Diese Parameter können im JSON-Format in dem Feld angegeben werden.
```

> Beispielsweise erwartet das Gateway einen Bestellwert, der als Parameter in einem JSON-Format angegeben wird:

Author: ```suggestion
In diesem konkreten Beispiel wird der Parameter folgendermaßen angegeben:
```

```json
{
  "shoppingCardAmount": 100
}
```

> Mit dem Wert von mindestens 100 EUR wird der Prozess durch den ersten Sequenzfluss fortgesetzt. Andernfalls wird der Pfad genommen, der durch den zweiten Squenzfluss folgt.

Author: ```suggestion
Mit dem Wert von mindestens 100 EUR wird der Prozess durch den unteren Sequenzfluss fortgesetzt. Andernfalls wird der Pfad genommen, der durch den oberen Squenzfluss folgt.
```

Die Steuerung des Diagramms aus einem Skript heraus ist denkbar einfach.

> Wie weiter oben bereits angedeutet, lassen sich Prozesse nach dem Deployment (das muss mit BPMN Studio erfolgen) mit Hilfe des [`ProcessEngineClient`](https://github.com/process-engine/getting-started/tree/develop/DotNet) starten:

Author: ```suggestion
Wie weiter oben bereits angedeutet, lassen sich Prozesse nach dem Deployment (das muss mit BPMN Studio erfolgen) mit Hilfe des [`ProcessEngineClient`](https://github.com/process-engine/getting-started/tree/develop/DotNet/CreateProcessInstance) starten:
```

    }
}
```

> Standardmäßig wird der Prozess auf Port 56000 gestartet. Dieser Port steht für die stable-Version, der für den Kunden bereitgestellt ist. Der Port 56001 ist für die Vorab-beta-Version und dient zu der technischen Umsetzung. Der Port 56002 steht für die alpha-Version, welche die neusten Funktionalitäten zur Erprobung beinhaltet. 

Author: ```suggestion
Standardmäßig wird der Prozess auf Port 56000 gestartet. Dieser Port steht für die stable-Version der ProcessEngine, die als offiziellen "Release" bereitgestellt ist. Der Port 56001 ist für die Beta-Version, die zwar schon in seinen Grundfunktionen nutzbar ist, jedoch noch nicht vollständig auf Fehler getestet worden ist. Der Port 56002 wird in der Alpha-Version genutzt, welche die neusten Funktionalitäten zur experimentellen Erprobung beinhaltet. 
```

    }
}
```
Der Port ist für die Instanz der ProcessEngine, die im BPMN Studio gestartet wird:

> * mit der reguläre "stable" Version startet das Studio auf Port 56000 eine ProcessEngine-Server-Instanz in neuster stabiler Version.

Author: ```suggestion
* Mit der regulären "stable" Version startet das Studio auf Port 56000 eine ProcessEngine-Server-Instanz in der aktuell stabilen Version.
```

> * Mit der *Beta*-Version startet das Studio auf Port 56100 eine ProcessEngine-Server-Instanz in neuster *Beta*-Version.

Author: ```suggestion
* Mit der *Beta*-Version startet das Studio auf Port 56100 eine ProcessEngine-Server-Instanz in der aktuellen *Beta*-Version.
```

    }
}
```

> Standardmäßig wird der Prozess auf Port 56000 gestartet. Dieser Port steht für die stable-Version der ProcessEngine, die als offiziellen "Release" bereitgestellt ist. Der Port 56001 ist für die Beta-Version, die zwar schon in seinen Grundfunktionen nutzbar ist, jedoch noch nicht vollständig auf Fehler getestet worden ist. Der Port 56002 wird in der Alpha-Version genutzt, welche die neusten Funktionalitäten zur experimentellen Erprobung beinhaltet. 

Author: ```suggestion
Standardmäßig wird der Prozess auf Port 56000 gestartet. Dieser Port steht für die stable-Version der ProcessEngine, die als offizielles "Release" bereitgestellt wird. Der Port 56001 ist für die Beta-Version, die zwar schon in seinen Grundfunktionen nutzbar ist, jedoch noch nicht vollständig auf Fehler getestet worden ist. Der Port 56002 wird in der Alpha-Version genutzt, welche die neusten Funktionalitäten zur experimentellen Erprobung beinhaltet. 
```

> Standardmäßig wird der Prozess auf Port 56000 gestartet. Dieser Port steht für die stable-Version der ProcessEngine, die als offizielles "Release" bereitgestellt wird. Der Port 56001 ist für die Beta-Version, die zwar schon in seinen Grundfunktionen nutzbar ist, jedoch noch nicht vollständig auf Fehler getestet worden ist. Der Port 56002 wird in der Alpha-Version genutzt, welche die neusten Funktionalitäten zur experimentellen Erprobung beinhaltet. 

Reviewer: Hier sind zwei Dinge inhaltlich nicht richtig: Bei den Ports hatte ich mich missverständlich ausgedrückt, es sind `56000`, `56100` und `56200`.

Der Port ist für die Instanz der ProcessEngine, die im BPMN Studio gestartet wird: 

- Starte ich die reguläre, "stable" Version, so startet das Studio auf Port 56000 eine ProcessEngine-Server-Instanz in neuster stabiler Version.
- Starte ich die Beta-Version, so startet das Studio auf Port 56100 eine ProcessEngine-Server-Instanz in neuster *Beta*-Version.

Die Alpha-Versionen hier bitte nicht erwähnen, da sie auch bewusst nicht unter "Downloads" verlinkt sind.
